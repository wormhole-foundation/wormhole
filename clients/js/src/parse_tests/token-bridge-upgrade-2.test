010000000100000000005c14b8e300010000000000000000000000000000000000000000000000000000000000000004fd81c1b1836cc25620000000000000000000000000000000000000000000546f6b656e4272696467650200030000000000000000000000000000000000000000000000000000000000000fb2
