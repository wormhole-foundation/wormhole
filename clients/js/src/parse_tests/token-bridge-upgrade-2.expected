{
  version: 1,
  guardianSetIndex: 1,
  signatures: [],
  timestamp: 0,
  nonce: 1544861923,
  emitterChain: 1,
  emitterAddress: '0x0000000000000000000000000000000000000000000000000000000000000004',
  sequence: 18267094531749757526n,
  consistencyLevel: 32,
  payload: {
    module: 'TokenBridge',
    type: 'ContractUpgrade',
    chain: 3,
    address: '0x0000000000000000000000000000000000000000000000000000000000000fb2'
  },
  digest: '0x7c8bd53e23a704a5476810d36335d2b9d65e4182e4863af3b27bd6a1ac8825e4'
}
